lib/Settings/Navigation.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:cboard_mobile/shared/app-bar.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class Navigation extends StatelessWidget {
```

Comment:
Take a look at some of the comments I made in `Display` and see if you can apply them here!